opm/input/eclipse/Schedule/GasLiftOpt.cpp

```diff
@@ -39,6 +67,34 @@ bool GasLiftGroup::operator==(const GasLiftGroup& other) const
            this->m_max_total_gas == other.m_max_total_gas;
 }
 
+// ---------------------------------------------------------------------------
+
+GasLiftWell::GasLiftWell(const RestartIO::RstWell& rst_well)
+    : m_name            { rst_well.name }
+    , m_max_rate        { liftOptMaxRateIfDefined(rst_well.glift_max_rate) }
+    , m_min_rate        { rst_well.glift_min_rate }
+    , m_use_glo         { rst_well.glift_active }
+    , m_weight          { rst_well.glift_weight_factor }
+    , m_inc_weight      { rst_well.glift_inc_weight_factor }
+    , m_alloc_extra_gas { rst_well.glift_alloc_extra_gas }
+{}
+
+// Unfortunately it seems just using the rst_well.glift_active flag is not
+// sufficient to determine whether or not the well should be included in gas
+// lift optimization.  The current implementation based on numerical values
+// found in the restart file is pure guesswork.
+bool GasLiftWell::active(const RestartIO::RstWell& rst_well)
+{
+    if (rst_well.glift_max_rate +
```

Comment:
could be return a+b+c != 0

Reply:
> could be return a+b+c != 0

Good point.  I've pushed an update to that effect.  It's not unlikely, though, that we'll revise this condition in the future.